docs/scripts/pull_from_json.py

```diff
@@ -0,0 +1,57 @@
+"""Script to pull from the hosted JSON and populate collection information."""
+import os
+import requests
+
+# URL with the hosted JSON
+url = 'https://climateknowledgeportal.worldbank.org/themes/cckp/data/catalog/tree_all.json'
+
+# read json from url
+r = requests.get(url)
+data = r.json()
+
+# plan to make an md file for each top level dictionary
+for collection in data.keys():
+    # delete file if it exists
+    if os.path.exists('docs/collections/' + collection + '.md'):
+        os.remove('docs/collections/' + collection + '.md')
+    # create file for writing
+    f = open('docs/collections/' + collection + '.md', 'w')
```

Comment:
@elbeejay If `docs/collections/` doesn't exist, it will raise an error.

Reply:
Right - the aim is to ensure that the documentation is always up-to-date, so by not commiting the subdirectory to the repository and removing it, we can have confidence that if the documentation builds, the latest JSON file was fetched and these pages were generated.

I figured it was a safe way to make sure we didn't end up with stale pages if or as collections are edited and modified.